src/CodeCoverageExtension.php

```diff
@@ -12,18 +10,24 @@
  * that was distributed with this source code.
  */
 
+declare(strict_types=1);
+
 namespace FriendsOfPhpSpec\PhpSpec\CodeCoverage;
 
 use FriendsOfPhpSpec\PhpSpec\CodeCoverage\Exception\NoCoverageDriverAvailableException;
 use FriendsOfPhpSpec\PhpSpec\CodeCoverage\Listener\CodeCoverageListener;
 use PhpSpec\Extension;
 use PhpSpec\ServiceContainer;
+use RuntimeException;
 use SebastianBergmann\CodeCoverage\CodeCoverage;
 use SebastianBergmann\CodeCoverage\Filter;
 use SebastianBergmann\CodeCoverage\Report;
 use SebastianBergmann\CodeCoverage\Version;
 use Symfony\Component\Console\Input\InputOption;
 
+use function count;
+use function is_array;
```

Comment:
Why do we need to explicitly import those functions? Are they overwritten somewhere? :thinking:

Reply:
PHPCsFixer from GrumpPHP does this.